Aged balance report: the only sensible filter is by end period

guewen opened the pull request:
The filters by date or by start period are meaningless for an aged partner
balance. Hide the choice of filters and let only the 'end period' available.

Commit:
Aged balance report: the only sensible filter is by end period

The filters by date or by start period are meaningless for an aged partner
balance. Hide the choice of filters and let only the 'end period' available.

pedrobaeza commented on account_financial_report_webkit/i18n/es.po:
```diff
@@ -1367,3 +1367,8 @@ msgstr "¡Valor haber o debe erróneo en el asiento contable!"
+#. module: account_financial_report_webkit
+#: view:account.aged.trial.balance.webkit:account_financial_report_webkit.account_aged_trial_balance_webkit
+msgid "At the end of"
+msgstr ""
```
You can put `Al final de`

guewen replied:
done! thanks

pedrobaeza commented:
Little improvement, the rest :+1:

Commit:
Add es translation term

sbidoul commented:
@guewen thanks for this contrib. A real usability improvement!

There seems to be a genuine test failure, though: except_orm: ('Error!', 'Select a starting and an ending period.')

Commit:
Remove the dates and "period from" from the aged balance tests

Commit:
Set a first period found because the start period is mandatory.

And what we want is to look in all the periods since the beginning.
Also set a default fiscal year to the current one so we have a default
end period.

Commit:
The end period must be set and thus the fiscal year too

Commit:
Set the required period to

Commit:
Remove contradictory help message

guewen commented:
Wow, I missed a lot of things! I guess I was too hasty to leave for holidays :flushed:  Fixed now.

guewen commented:
Still an issue, fixing it.

Commit:
Change the periods when the fiscal year is modified

The default value on period_from is now useless because set by the
onchange when we set the fiscalyear.

guewen commented:
It should be ok now.

sbidoul commented:
@adrienpeiffer @chrisdec have you had a chance to test this one?

chrisdec commented:
:+1: (functional test)
